service/src/test/java/uk/gov/hmcts/reform/fpl/service/CafcassEmailContentProviderSDOIssuedTest.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.config.CafcassLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.CafcassLookupConfiguration.Cafcass;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.email.content.CafcassEmailContentProvider;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
+
+@ExtendWith(SpringExtension.class)
+class CafcassEmailContentProviderSDOIssuedTest {
+
+    private static final String LOCAL_AUTHORITY_CODE = "example";
+    private static final String CAFCASS_NAME = "Test cafcass";
+    private static final String COURT_EMAIL_ADDRESS = "FamilyPublicLaw+test@gmail.com";
+
+    @Mock
+    private CafcassLookupConfiguration cafcassLookupConfiguration;
+
+    @Mock
+    private LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+
+    @Mock
+    MapperService mapperService;
+
+    @Mock
+    DateFormatterService dateFormatterService;
+
+    @Mock
+    HearingBookingService hearingBookingService;
+
+    @InjectMocks
+    private CafcassEmailContentProvider cafcassEmailContentProvider;
+
+    @BeforeEach
+    void setup() {
+        given(mapperService.mapObject(Mockito.any(), Mockito.any()))
```

Comment:
I wrote a comment about mocking mapperService higher up I think, I don't see why we need to mock, should be able to use the real thing